content/docs/0.8.x/reference/cli/_index.md

```diff
@@ -198,3 +198,144 @@ The `KEPTNCONFIG` environment variable holds a config file that contains the cre
+
+### Install bash-completion for linux
+
+bash-completion is provided by many package managers (see [here](https://github.com/scop/bash-completion#installation)). You can install it with `apt-get install bash-completion` or `yum install bash-completion`, etc.
+
+The above commands create /usr/share/bash-completion/bash_completion, which is the main script of bash-completion. Depending on your package manager, you have to manually source this file in your ~/.bashrc file.
+
+To find out, reload your shell and run type _init_completion. If the command succeeds, you're already set, otherwise add the following to your ~/.bashrc file:
+
+```bash
+source /usr/share/bash-completion/bash_completion
+```
+
+Reload your shell and verify that bash-completion is correctly installed by typing type _init_completion.
+
+### Enable Keptn CLI autocompletion for linux (bash)
+
+You now need to ensure that the Keptn CLI completion script gets sourced in all your shell sessions. There are two ways in which you can do this:
+
+- source the completion script in your ~/.bashrc file:
+
+```bash
+echo 'source <(keptn completion bash)' >>~/.bashrc
+```
+
+- Add the completion script to the /etc/bash_completion.d directory:
+
+```bash
+keptn completion bash >/etc/bash_completion.d/keptn
+```
+
+If you have an alias for Keptn CLI, you can extend shell completion to work with that alias:
+
+```bash
+echo 'alias kc=keptn' >>~/.bashrc
+echo 'complete -F __start_keptn kc' >>~/.bashrc
+```
+
+After reloading your shell, Keptn CLI autocompletion will be enabled succesfully.
+</details>
+
+<details><summary>Autocompletion for Bash (MacOS)</summary>
+
+The Keptn CLI completion script for Bash can be generated with `keptn completion bash`. Sourcing this script in your shell enables Keptn CLI autocompletion.
+
+However, the Keptb CLI autocompletion script depends on bash-completion which you thus have to previously install.
+
+> Warning: There are two versions of bash-completion, v1 and v2. V1 is for Bash 3.2 (which is the default on macOS), and v2 is for Bash 4.1+. The Keptn CLI autocompletion script doesn't work correctly with bash-completion v1 and Bash 3.2. It requires bash-completion v2 and Bash 4.1+. Thus, to be able to correctly use the Keptn CLI autocompletion on macOS, you have to install and use Bash 4.1+ (instructions). The following instructions assume that you use Bash 4.1+ (that is, any Bash version of 4.1 or newer).
+
+### Install bash-completion for MacOS
+
+Note: As mentioned, these instructions assume you use Bash 4.1+, which means you will install bash-completion v2 (in contrast to Bash 3.2 and bash-completion v1, in which case Keptn CLI completion won't work).
+
+You can test if you have bash-completion v2 already installed with type _init_completion. If not, you can install it with Homebrew:
```

Comment:
Same as above, please change type _init_completion to `type _init_completion` (add single quotes)